src/ert/_c_wrappers/enkf/config/enkf_config_node.py

```diff
@@ -440,57 +440,30 @@ def __ne__(self, other):
         return not self == other
 
     def __eq__(self, other) -> bool:
-        if self.getImplementationType() != other.getImplementationType():
+        checks = [
+            self.getImplementationType() == other.getImplementationType(),
+            self.getKey() == other.getKey(),
+            self.get_init_file_fmt() == other.get_init_file_fmt(),
```

Comment:
Personally I would have used

```python
if any(
    self.getImplementationType() != other.getImplementationType(),
    self.getKey() != other.getKey(),
    self.get_init_file_fmt() != other.get_init_file_fmt(),
    self.get_enkf_outfile() != other.get_enkf_outfile(),
    self.getUseForwardInit() != other.getUseForwardInit(),
):
    return False
```

Note the missing `[` and `]` which makes it a generator so that the first element where there is inequality, we break and return False.

This is a irrelevant detail though, so take it or leave it as you see fit :)

Reply:
Thanks for feedback!